docs/content/latest/quick-start/build-apps/go/ysql-pq-ssl.md

```diff
@@ -0,0 +1,207 @@
+    </a>
+  </li>
+  <li >
+    <a href="../ysql-gorm/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YSQL - GORM
+    </a>
+  </li>
+  <li>
+    <a href="../ycql/" class="nav-link">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+</ul>
+
+The following tutorial creates a simple Go application that connects to a YugabyteDB cluster using the [pq driver](https://godoc.org/github.com/lib/pq), performs a few basic database operations — creating a table, inserting data, and running a SQL query — and then prints the results to the screen.
+
+## Prerequisites
+
+This tutorial assumes that:
+
+- YugabyteDB is up and running. If you are new to YugabyteDB, you can download, install, and have YugabyteDB up and running within minutes by following the steps in [Quick start](../../../../quick-start/). Alternatively, you can use [Yugabyte Cloud](http://cloud.yugabyte.com/register), to get a fully managed database-as-a-service (DBaaS) for YugabyteDB.
+
+- [Go version 1.8](https://golang.org/dl/), or later, is installed.
+
+### SSL/TLS configuration
+
+Refer to the [SSL/TLS configuration](../../../../secure/tls-encryption/client-to-server/) page to launch your YugabyteDB cluster with SSL/TLS enabled. This is already taken care of for you in **Yugabyte Cloud**, as the cluster is pre-configured to do client-side authentication with SSL/TLS enabled.
+
+### OpenSSL
+
+Install [OpenSSL](https://www.openssl.org/) 1.1.1 or later only if you have a YugabyteDB setup with SSL/TLS enabled. The **Yugabyte Cloud** clusters are always SSL/TLS enabled.
+
+### CA certificate
+
+Download the CA certificate file to allow your application connect securely to the YugabyteDB cluster. In case of a **Yugabyte Cloud** cluster,
+
+1. On the **Clusters** tab, select a cluster.
+
+1. Click **Connect**
+
+1. Click **Connect to your application** and download the CA cert.
+
+### Go PostgreSQL driver
+
+The [Go PostgreSQL driver package (`pq`)](https://godoc.org/github.com/lib/pq) is a Go PostgreSQL driver for the `database/sql` package.
+
+To install the package locally, run the following commands:
+
+{{< note title="Note">}}
+The environment variable `GO111MODULE` may need to be set, if your Go version is 1.11 or higher. It is set before installing the lib/pq package.
+{{< /note >}}
+
+```sh
+$ export GO111MODULE=auto
+$ go get github.com/lib/pq
+```
+
+## Create the sample Go application
+
+Create a file `ybsql_hello_world.go` and copy the contents below.
+
+{{< note title="Note">}}
+The constants defined under the `const` block in the code below have the default values.
+You may need to change the values of `host`, `user` and `password` as per your YugabyteDB cluster setup.
+For a **Yugabyte Cloud** cluster, the `host` will have a value in the format similar to `xxxx-xxxx-xxxx-xxxx-xxxx.aws.ybdb.io`.
+{{< /note >}}
+
+```go
+package main
+
+import (
+  "database/sql"
+  "fmt"
+  "log"
+
+  _ "github.com/lib/pq"
```

Comment:
what's that underscore? i see it is also in the ysql-pq file, looks like a typo

Reply:
That is the go syntax to use the package which we import . Not using the underscore gives an error:
`./ybsql_hello.go:8:3: imported and not used: "github.com/lib/pq"`

Reply:
This is a valid import. We're importing lib/pq package so that the driver gets registered for use via database/sql package. For the other two drivers, we use their APIs directly.